linux/netdev.c

```diff
@@ -146,3 +147,22 @@ bool netdev_set_up_promisc(const char *const ifname, bool up, bool promisc)
 	close(fd);
 	return true;
 }
+
+bool netdev_check_if_exists(const char *const ifname)
+{
+	struct ifaddrs *ifaddr, *ifa;
+	bool found = false;
+
+	if (getifaddrs(&ifaddr) == -1) {
+		printf("can not check interface names, continuing\n");
```

Comment:
Logging in libuwifi and horst ist done via the LOG_INF/ERR... macros and not via printf(). in this case LOG_WARN would be appropriate

Reply:
Done.